src/Parallafka/Parallafka.cs

```diff
@@ -155,6 +155,9 @@ private async Task KafkaPollerThread(ITargetBlock<IKafkaMessage<TKey, TValue>> r
                             }
                         }
                     }
+                    catch (OperationCanceledException)
+                    {
```

Comment:
Should this throw if `!stopToken.IsCancellationRequested`?

Reply:
The only way for OCE to be thrown is when PollAsync stops with an OCE, so I don't think so.

Reply:
I am looking deeper into the usage of the stopToken... more changes soon.

Reply:
I was just thinking in case something changes or we don't know the PollAsync implementation and it throws OCE for some other reason

Reply:
`IKafkaConsumer<TKey, TValue>` was updated to describe the correct pattern for using CancellationToken:

```c#
 /// <summary>
        /// Polls Kafka for the next record.
        /// Returns the next record, or throws an <see cref="OperationCanceledException"/> if the cancellationToken is cancelled.
        /// </summary>
```

See perhaps: https://devblogs.microsoft.com/premier-developer/recommended-patterns-for-cancellationtoken/